readthedocs/projects/forms.py

```diff
@@ -272,6 +273,69 @@ def __init__(self, *args, **kwargs):
         else:
             self.fields['default_version'].widget.attrs['readonly'] = True
 
+        self.setup_external_builds_option()
+
+    def setup_external_builds_option(self):
+        """Disable the external builds option if the project doesn't meet the requirements."""
+        msg = None
+        url = reverse("projects_integrations", args=[self.instance.slug])
+        integrations = list(self.instance.integrations.all())
```

Comment:
Why are you converting this to a list here?

Reply:
this saves one query later where we use `len(integations)` instead of `.count()`